JSDemos/Demos/DataGrid/GroupSummaries/description.md

```diff
@@ -1 +1,19 @@
-This example illustrates the use of&nbsp;group data summaries (calculated totals in&nbsp;each group). To&nbsp;configure a&nbsp;group summary, declare the **groupItems** array within the **summary** object. This demo also illustrates the component&rsquo;s ability to&nbsp;position group summary items. By&nbsp;default, they are displayed in&nbsp;the group row&rsquo;s header, but you can shift them to&nbsp;a&nbsp;specific column (the **alignByColumn** property) or&nbsp;even move them to&nbsp;the group footer (the **showInGroupFooter** property).
\ No newline at end of file
+A group summary creates a synopsis for records in a group. To configure this feature, populate the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[groupItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/) array with summary configuration objects. Each object should specify a [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) that supplies data for summary calculation and a [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType) &mdash; an aggregate function that should be applied to this data.
+
+### Position a Group Summary
+
+The default position for group summaries are the parentheses after the group caption. You can change the position to one of the following:
+
+DataGrid will use the value from the column property of the summary. 
+
+* Group row's column
+To position a summary in the group row, set the [alignByColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#alignByColumn) property to **true**. DataGrid will position the summary in the column specified in the summary's **column** property. If you want to use another column, specify it in the [showInColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInColumn) property.
+
+* Group footer
+To position a summary in the group footer, set the [showInGroupFooter](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInGroupFooter) property to **true**. DataGrid aligns summaries to the right of the footer. If you want to display a summary in a certain column of the footer, specify it in the [showInColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInColumn) property.
+
+### Format Summary Values
+To set a text for the summary value, specify it in the [displayFormat](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#displayFormat) property.
```

Comment:
What about the `valueFormat`?
Don't forget to make references to the demo itself so that the reader knows where to look in the code.